docs/ansible-release-announcement/major-release/announcement-email.md

```diff
@@ -0,0 +1,83 @@
+Release announcement : Ansible Community Package X.0.0
+
+Hello everyone,
+
+We're happy to announce the release of the Ansible X.0.0 package!
+
+Ansible X.0.0 requires latest version of ansible-core a.b and includes a curated set of Ansible collections that provides a vast number of modules and plugins. This is a major release of Ansible.
+
+ansible-core is a required dependency, not contained within the ansible packages. Pip install builds the
+dependency, but it can be built and installed quite independently of the "ansible" distribution.
```

Comment:
The part "pip install builds the dependency" is wrong in general. ansible-core has wheels, so in general nothing needs to be built.